### PR TITLE
RATEST-242 : Fix ignored Conditions Feature file

### PR DESCRIPTION
**Pull request by mherman22**
This is an attempt to fix -> https://issues.openmrs.org/browse/RATEST-242 here https://github.com/openmrs/openmrs-contrib-qaframework/blob/1594bc4349db0b385d4469aaf6e2eff895359168/src/test/resources/features/refapp-2.x/12-condition/conditions.feature

### CODE REVIEW

File: src/test/resources/features/refapp-2.x/12-condition/conditions.feature [hunk 1]
```diff
@@ -16,27 +16,32 @@ Feature: Patient's Conditions Managment
...
       | inactive |
-
+	
+	@selenium
   @dashboard
   Scenario: Set first condition to inactive
     And User clicks on set inactive button
     Then System should move condition to inactive section
-
+
```
[the remark this answers is not in the document]
Comment (mherman22) [reply src/test/resources/features/refapp-2.x/12-condition/conditions.feature:25]: hey @sherrif10 , by `cross check all the feature files` do you mean all the feature files in `refapp2.x`? or only limited to `conditions.feature`

[the remark this answers is not in the document]
Comment (mherman22) [reply src/test/resources/features/refapp-2.x/12-condition/conditions.feature:25]: hey @sherrif10 , by `cross check all the feature files` do you mean all the feature files in `refapp2.x`? or only limited to `conditions.feature`

[the remark this answers is not in the document]
Comment (mherman22) [reply src/test/resources/features/refapp-2.x/12-condition/conditions.feature:25]: hey @sherrif10 , by `cross check all the feature files` do you mean all the feature files in `refapp2.x`? or only limited to `conditions.feature`

[the remark this answers is not in the document]
Comment (sherrif10) [reply src/test/resources/features/refapp-2.x/12-condition/conditions.feature:25]: Check conditions.feature and check whether the scenarios written in correspond to conditionsSteps class

[the remark this answers is not in the document]
Comment (mherman22) [reply src/test/resources/features/refapp-2.x/12-condition/conditions.feature:25]: ooh okay let me do that

[the remark this answers is not in the document]
Comment (mherman22) [reply src/test/resources/features/refapp-2.x/12-condition/conditions.feature:25]: i have checked through and the `conditionsSteps` class is up-to-date with `conditions.feature` feature file

**Comment by sherrif10** [comment]
Please fix the failures ,Try to follow the logs for quick direction on where the cause comes from

**Comment by mherman22** [comment]
> Please fix the failures ,Try to follow the logs for quick direction on where the cause comes from

The failures look to be coming from 
`  Set first condition to active(Patient's Conditions Managment): Expected condition failed: waiting for visibility of element located by By.xpath: //button[text()='Set Active'] (tried for 120 second(s) with 500 milliseconds interval)(..)
  Edit first active condition(Patient's Conditions Managment): Expected condition failed: waiting for org.openmrs.uitestframework.page.Page$1@3837d7cc (tried for 120 second(s) with 500 milliseconds interval)(..)
  Edit first inactive condition(Patient's Conditions Managment): Expected condition failed: waiting for org.openmrs.uitestframework.page.Page$1@2ef76846 (tried for 120 second(s) with 500 milliseconds interval)(..)
  Delete first condition(Patient's Conditions Managment): Element <button id="confirmIdentifierId" class="confirm right"> could not be scrolled into view(..)` and i do't quite understand why. 
Does it have something to do with this repo not being connected to the **uitestframework** (anymore)?

**Comment by sherrif10** [comment]
Add a waiting element on this line   https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java#L54    following the logs from the problem is coming from this line  https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java#L107

So create a simple pull request that adds an element for visibility of an element ,[ i did something similar here ](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/516/files)

**Comment by sherrif10** [comment]
Hello did you get a chance to look through this wiki document https://wiki.openmrs.org/display/docs/Pull+Request+Tips  . We must ensure we follow openmrs standard. the branch name of this ticket doesn't follow openmrs standards  please

**Comment by mherman22** [comment]
> Hello did you get a chance to look through this wiki document https://wiki.openmrs.org/display/docs/Pull+Request+Tips . We must ensure we follow openmrs standard. the branch name of this ticket doesn't follow openmrs standards correct that as well

thank you, i didnt get chance to. Let me do that commit better changes

**Comment by mherman22** [comment]
let me create a new PR. thanks

**Comment by mherman22** [comment]
new pr created at https://github.com/openmrs/openmrs-contrib-qaframework/pull/141

**Pull request closed without merging**